Tasks/KubernetesManifestV0/src/actions/deploy.ts

```diff
@@ -1,47 +1,123 @@
 "use strict";
 
-import tl = require('vsts-task-lib/task');
-import path = require('path');
 import fs = require("fs");
+import path = require('path');
+import tl = require('vsts-task-lib/task');
+import yaml = require('js-yaml');
+import * as canaryDeploymentHelper from '../CanaryDeploymentHelper';
+import * as KubernetesObjectUtility from '../KubernetesObjectUtility';
 import * as utils from "./../utilities";
 import { IExecSyncResult } from 'vsts-task-lib/toolrunner';
 import { Kubectl, Resource } from "utility-common/kubectl-object-model";
 
+const TASK_INPUT_NAMESPACE = "namespace";
+const TASK_INPUT_CONTAINERS = "containers";
+const TASK_INPUT_CANARY_PERCENTAGE = "percentage";
+const TASK_INPUT_DEPLOYMENT_STRATEGY = "deploymentStrategy";
```

Comment:
Is this required? `TASK_INPUT_*`
If you think this will be used multiple times, why don't we move this to a file called `TaskInputParameters` or something, access all the inputs there and use the values from that model?